src/main/scala/nl.knaw.dans.easy.sword2/DepositHandler.scala

```diff
@@ -126,7 +126,11 @@ object DepositHandler {
   }
 
   private def extractBag(mimeType: String)(implicit settings: Settings, id: String): Try[File] = {
-    def extract(file: File, outputPath: String): Unit = new ZipFile(file.getPath).extractAll(outputPath)
+    def extract(file: File, outputPath: String): Unit = {
+      val zip = new ZipFile(file.getPath)
```

Comment:
General tip on API's that require you to call extra setters after the constructor call: expand the constructor as shown below. I'm not saying you should use this in this piece of code; it's more a general remark. I've been using this trick when I need to call a constructor, call some setters and return the object afterwards.

```
new ZipFile(file.getPath) {
    setFileNameCharset(StandardCharsets.UTF_8.name())
}
```